.github/workflows/build.yml

```
@@ -85,7 +143,10 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       contents: write
-    needs: build
+    needs: 
+      - build
+      - verifyNonMatching
+      - generatePreviousReport
```

```suggestion
      - build
      - build-previous
```
website doesn't technically rely on anything from the non-matching build